db/structure.sql

```diff
@@ -2,11 +2,12 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.6
```

Comment:
Please remove the changes to this file from the PR as they are just local noise from your environment and not part of the change.